core/data/coercion.go

```diff
@@ -50,6 +50,8 @@ func CoerceToString(val interface{}) (string, error) {
 		return t, nil
 	case int:
 		return strconv.Itoa(t), nil
+	case int64:
```

Comment:
@mellistibco what should int64 map to of our types?  Should we have int, int64, float, float64 types?  Also what should the types be Integer, Integer64, Float32, Float64?  Or Integer, Long, Float, Double?  Int's size is machine dependent, but there is no machine dependent float.  The general suggestion is to use int and float64 in golang code.  The reason we just had int & float64 was because of that.  @mellistibco thoughts?

Reply:
@fm-tibco For WI we use it most on function part. we allow function access input or return for int64. 
As WI case,  we using complex_object and the type of each field control by JSON schema IN UI and we do not restrict the type in runtime to let Go cast it on the fly,  Go reflect usually return a number as int64 so we use it.

Reply:
@fm-tibco  I think we should follow current OSS way, for now, to keep `int` as it is machine dependent. I will make this change.